RocketChat/Rocket.Chat.ReactNative
# Chore: Change console.log to console.error on log()

## Proposed changes

Log red screen error as `console.error` instead `console.log` and show the error properly

## Issue(s)	


## How to test or reproduce


This test plan is to developers:
- Open the Reactotron and connect with the emulator
- Go to https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/782bdda2c7d8bc77d390d9549fb24302f4cc9804/app/containers/EmojiPicker/index.tsx#L88
- Insert a code as `const e = emoji.content.test.emoji` and save
- Then open the `open.rocket.chat`, inside any room, do a long press on any message, click on emoji with plus signal, and tap on any emoji.
- On Reactotron will appear a line with `{ }` - this is the error returned, but Reactotron can't understand
- Then, use this branch, do the same steps, and will alert on Reactotron something with `ERROR`

## Screenshots

#### Before
<img width="907" alt="Screen Shot 2022-01-26 at 00 19 10" src="https://user-images.githubusercontent.com/47038980/151098412-0f5c32f3-f20d-45eb-99c6-1707635e47d8.png">


#### After
<img width="907" alt="Screen Shot 2022-01-26 at 00 17 38" src="https://user-images.githubusercontent.com/47038980/151098329-0f847b4a-d7eb-4185-b87e-53d627b1912f.png">



## Types of changes




## Checklist



## Further comments
